_js/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
I'm not sure I understand what this file does. Please document it.

Reply:
Testing browser usage, or should I rename this file?